### DISCUSSION TITLE
How to edit text of a PSD file?

### DISCUSSION THREAD

**Discussion by failable** [EmilDohne/PhotoshopAPI#125] [Q&A] [answered]
The title says it all...

**Comment by EmilDohne** [comment, marked as answer]
Hey @failable, I'm assuming you're talking about text layers? These are unfortunately not implemented yet but will come in a future update. I've got most of the framework in place to actually modify them though so once I get around to it implementation shouldn't be too difficult

**Reply by EmilDohne** [reply]
I've gone ahead and create a ticket to track the progress of this:

#126
